Cargo.toml

```diff
@@ -15,11 +15,12 @@ exclude = [
 ]
 
 [dependencies]
-plist = "0.4"
-serde = "1.0"
+plist = "0.5"
+serde = { version =  "1.0", features = ["rc"] }
 serde_derive = "1.0"
 serde_repr = "0.1"
 quick-xml = "0.12.0"
+druid = { git="https://github.com/xi-editor/druid/", rev="8164adf", optional=true }
```

Comment:
Is this for the Data trait? Seems *almost* like a wrong-way dep...

Hmm, now I see, also druid's piet's color type.

Reply:
and kurbo's affine, want to have it pinned to druid instead of those crates specifically. 🤷‍♂